src/luminol/algorithms/correlator_algorithms/cross_correlator.py

```diff
@@ -9,6 +9,7 @@
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 """
+from builtins import map
```

Comment:
Looks like the only use of `map` is Line 91; if you use a list comprehension, or if you wrap the `map` in a `list()` call, is there any need for this import?

Reply:
If we transition the list/lambda/map statements to list comprehensions then we shouldn't need this.